recipes-openxt/xenclient/xenclient-toolstack_git.bb

```diff
@@ -1,7 +1,7 @@
 DESCRIPTION = "XenClient toolstack"
 LICENSE = "LGPLv2.1"
 LIC_FILES_CHKSUM = "file://COPYING;md5=321bf41f280cf805086dd5a720b37785"
-DEPENDS += "ocaml-cross ocaml-dbus ocaml-camomile xen xz"
+DEPENDS += " ocaml-cross ocaml-dbus ocaml-camomile xen xen-libxl xz "
```

Comment:
leading and trailing spaces are not required ([+=/=+, appending with spaces](https://www.yoctoproject.org/docs/1.6/bitbake-user-manual/bitbake-user-manual.html#appending-and-prepending)).